File: HACKATHON/js/loan.js
// Loan EMI Calculator
function calculateEMI(principal, rate, months) {
  let monthlyRate = rate / 12 / 100;
  let emi = (principal * monthlyRate * Math.pow(1 + monthlyRate, months)) / (Math.pow(1 + monthlyRate, months) - 1);
  return emi.toFixed(2);
}

function showLoan(principal, rate, months) {
  let emi = calculateEMI(principal, rate, months);
  document.getElementById("content").innerHTML = `<h3>Loan Planning</h3><p>Loan: ₹${principal}<br>Rate: ${rate}%<br>Months: ${months}<br>EMI: ₹${emi}</p>`;
}

File: HACKATHON/java/Loan.java
public class Loan {
    private double principal;
    private double rate;
    private int months;

    public Loan(double principal, double rate, int months) {
        this.principal = principal;
        this.rate = rate;
        this.months = months;
    }

    public double calculateEMI() {
        double monthlyRate = rate / 12 / 100;
        return (principal * monthlyRate * Math.pow(1 + monthlyRate, months)) /
               (Math.pow(1 + monthlyRate, months) - 1);
    }

}
